rest/blip_api_test.go

```diff
@@ -2752,15 +2752,15 @@ func TestBlipDeltaSyncNewAttachmentPull(t *testing.T) {
 		// Check the request body was the actual delta
 		msgBody, err := msg.Body()
 		assert.NoError(t, err)
-		assert.Equal(t, `{"_attachments":[{"hello.txt":{"digest":"sha1-Kq5sNclPz7QV2+lfQIuc6R7oRu0=","length":11,"revpos":2,"stub":true}}]}`, string(msgBody))
+		assert.Equal(t, `{"_attachments":[{"hello.txt":{"digest":"sha1-Kq5sNclPz7QV2+lfQIuc6R7oRu0=","length":11,"revpos":2,"stub":true,"ver":2}}]}`, string(msgBody))
```

Comment:
Do we need to send `"ver":2` to Couchbase Lite? Or is this an internal SG implementation detail?